lang/py/src/avro/schema.py

```diff
@@ -43,9 +43,17 @@
 import json
 import math
 import sys
+from warnings import warn
 
 from avro import constants
 
+#
+# Base class for warnings.
+#
+
+class AvroWarning(UserWarning):
```

Comment:
Consider making that comment above a docstring for this class. (Then remove the pass.)

Reply:
I also moved both warning classes beside the existing exceptions.